app/views/guide/process_application.html.slim

```diff
@@ -269,7 +272,32 @@ header
           span.hint
             |&nbsp;(not Advice and Assistance or Advice by Way of Representation)
     br
+    h5 If you get an unexpected benefits result  
+    br It takes a couple of days for the Department for Work and Pensions to be updated with information about new applicants. 
+    br If you’re processing the application at the counter, ask the applicant for a letter to show they’re receiving these benefits. 
+    br
+    h5 Check the name of the benefit 
+    br There are some benefits that are similarly named to the 6 qualifying benefits. If an applicant is receiving one of these benefits they should apply on the basis of their income. They are: 
+    ul 
```

Comment:
Some syntax errors found:
- 'Span' is displayed after Contribution based ESA
- 'Contribution-based' instances should have hyphen to be consistent with 'Income-based'
- ESA missing it's hint text in brackets
- Are 'Pension Credit' and 'Savings Credit' 2 separate bullets?
- Universal Credit line re: £6000 should be in brackets and hint text
- (ABWOR) should be hint text